Support/bone101/UI/bbui.js

```diff
@@ -909,6 +915,144 @@ var UI = (function() {
             return graph;
         })();
 
+        //graph global variables
+        var xWidth = 360;
```

Comment:
These should likely be organized as an object, rather than a large number of individual variables. In this reading, I cannot tell where these variables are being used.